ops/pebble.py

```diff
@@ -1205,9 +1232,20 @@ def _merge(self, other: Check):
         For attributes present in both objects, the passed in check
         attributes take precedence.
         """
+        # wouldn't it be better to use other.to_dict?
+        # it would let us skip these checks
         for name, value in other.__dict__.items():
             if not value or name == 'name':
                 continue
+            # how to handle CheckLevel.UNKNOWN?
+            #
+            # it would be skipped here if we used other.to_dict
+            #
+            # previously it would have been <some-unknown-value-string>
+            # while we might want <some-unknown-value-string> to overwrite,
+            # would we ever want Checklevel.UNKNOWN to overwrite?
+            if value is CheckLevel.UNKNOWN:
+                continue
```

Comment:
I think we should just use `other.to_dict()` here, is there a reason we weren't already?